src/content/overview/architecture/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Architecture
+description: High overview about how our platform is built and how you can interact with it.
```

Comment:
```suggestion
description: High level technical overview of our platform, its components and how you can interact with it.
```

Reply:
yeah that is a good point, let's be consistent